pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "foursight_core"
-version = "0.4.2"
+version = "0.4.3.0b0"
```

Comment:
I'd have use 0.4.2.1b0.

(You want something higher than 0.4.2 and not higher than any legit next version on master.)